common/components/common/groups/IframeGroupDisplay.jsx

```diff
@@ -0,0 +1,35 @@
+// @flow
+// derived from AboutGroupDisplay
+import React from "react";
+import { Container } from "flux/utils";
+import AboutGroupDisplay from "./AboutGroupDisplay.jsx"
+import type { GroupDetailsAPIData } from "../../utils/GroupAPIUtils.js";
+import ProjectCardsContainer from "../../componentsBySection/FindProjects/ProjectCardsContainer.jsx";
+
+class IframeGroupDisplay extends AboutGroupDisplay<Props, State> {
```

Comment:
Why does this need to inherit from AboutGroupDisplay?

Reply:
This function needs to have its own unique name, and would have a lot of code duplicated from AboutGroupDisplay, but the render function is different.  Doing it this way eliminates duplicate code.